docs/decisions/2022-04-website-testing.md

```diff
@@ -0,0 +1,39 @@
+# Testing the Website
```

Comment:
Please use the [template](https://github.com/GoogleCloudPlatform/emblem/blob/main/docs/decisions/template.md)

Reply:
Done.